doc/Programming-guide/Key-platform-features/Configuration-management/index.md

```diff
@@ -5,51 +5,63 @@ permalink: /:path/
 sort_idx: 60
 ---
 
-# Configuration
-
-- [Configuration management overview](#configuration-management-overview)
-- [Configuration schema](#configuration-schema)
-- [Addressing](#addressing)
-- [Records auto-generation](#records-auto-generation)
-- [Schema-specific configuration management](#schema-specific-configuration-management)
-  - [Base schema](#base-schema)
-  - [Updating configuration data](#updating-configuration-data)
-  - [UUID validation](#uuid-validation)
-    - [Updating an existing object (there is a UUID in the previous version of the configuration)](#updating-an-existing-object-there-is-a-uuid-in-the-previous-version-of-the-configuration)
-    - [Creating a new object (there is no UUID in previous version of the configuration)](#creating-a-new-object-there-is-no-uuid-in-previous-version-of-the-configuration)
-- [Group-specific configuration management](#group-specific-configuration-management)
-  - [Override schema](#override-schema)
-  - [Override algorithm](#override-algorithm)
-- [User-specific configuration management](#ser-specific-configuration-management)
-- [Endpoint data synchronization](#endpoint-data-synchronization)
-  - [Protocol schema](#protocol-schema)
-  - [Delta calculation algorithm](#delta-calculation-algorithm)
-- [Schema versioning](#schema-versioning)
-
-
-The Kaa Configuration subsystem is responsible for configuring endpoints by supplying them with the structured data of user-defined complexity that is managed via the Kaa server. The fact that Kaa operates with the uniformly structured data deserves a special emphasis. By knowing the data structure of the application, Kaa provides a number of very useful features, such as:
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Kaa Configuration subsystem is responsible for configuring endpoints by supplying them with the structured data of user-defined complexity that is managed
+via the Kaa server. The fact that Kaa operates with the uniformly structured data deserves a special emphasis. By knowing the data structure of the
+application, Kaa provides a number of very useful features, such as:
 
 * Incremental data updates for endpoints
 * Endpoint-specific data view that is based on the endpoint membership in endpoint groups
```

Comment:
This is not clear to me. Let's move this to the end of the list. Maybe put a bullet about support of versioning.